src/index.css

```diff
@@ -0,0 +1,593 @@
+}
+
+.label3 {
+  grid-area: l3;
+}
+
+.input3 {
+  grid-area: i3;
+}
+
+.label4 {
+  grid-area: l4;
+}
+
+.input4 {
+  grid-area: i4;
+}
+
+.label5 {
+  grid-area: l5;
+}
+
+.input5 {
+  grid-area: i5;
+}
+
+.label6 {
+  grid-area: l6;
+}
+
+.textarea {
+  grid-area: ta;
+  padding: 50px;
+  border: 1px solid var(--colour-sage);
+  border-radius: 5px;
+}
+
+.checkbox {
+  display: flex;
+  flex-direction: column;
+}
+
+.box {
+  margin-left: 30px;
+}
+
+.button-submit {
+  text-align: center;
+}
+
+.btn-submit {
+  background-color: var(--colour-sage);
+  color: var(--colour-white);
+  margin: 20px 0px 40px 0px;
+  padding: 10px 30px;
+  border: 0ch;
+}
+
+label {
+  color: var(--colour-sage);
+}
+
+input {
+  border: 1px solid var(--colour-sage);
+  border-radius: 5px;
+  padding: 10px;
+}
+
+/* .main-div {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+} */
+
+/* reservation menu here  */
+.menu-items {
+  background-image: url("./images/Reservations/Background.png");
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--colour-white);
+  min-height: 500px;
+  background-size: cover;
+}
+
+.menu-btn {
+  color: var(--colour-sage);
+  padding: 15px;
+}
+
+.img2-div {
+  display: none;
+}
+
+/* ----------Media query section starts here-------- */
+/* ----------- media query for tablet------*/
+@media only screen and (min-width: 400px) {
+
+  /* ------about section----- */
+  .blend-container {
+      display: flex;
+      flex-direction: row;
+      margin-top: 30px;
+ }
+
+
+  /* ----------wiseleaf grid for tablet------ */
+  .wiseleaf-container {
+    background-color: var(--colour-sage);
+    display: grid;
+    grid-template-columns: 40px 1fr 50px 2fr 40px;
+    grid-template-rows: 40px 2fr 50px 40px;
+    margin-bottom: 5em;
+  }
+
+  .wiseleaf-img1 {
+    grid-column: 2/3;
+    grid-row: 2/4;
+    /* width: 100%;
+    height: auto;
+    object-fit: cover; */
+    /* background-image: cover; */
+  }
+
+  .wiseleaf-info {
+    grid-column: 4/5;
+    grid-row: 1/3;
+  }
+
+  .wiseleaf-img2 {
+    grid-column: 4/5;
+    grid-row: 3/5;
+  }
+
+  /* ..........reservation-form for tablet..... */
+
+  .form-container {
+    display: grid;
+    grid-template-columns: repeat(2,80px);
+    grid-template-areas:
+      "l1 l1 l2 l3"
+      "i1 i1 i2 i3"
+      "l4 l4 l5 l5"
+      "i4 i4 i5 i5"
+      "l6 . . ."
+      "ta ta ta ta";
+    grid-gap: 5px;
+    margin: 2em;
+  }
+
+  .label1 {
+    grid-area: l1;
+  }
+
+  .input1 {
+    grid-area: i1;
+  }
+
+  .label2 {
+    grid-area: l2;
+  }
+
+  .input2 {
+    grid-area: i2;
+  }
+
+  .label3 {
+    grid-area: l3;
+  }
+
+  .input3 {
+    grid-area: i3;
+  }
+
+  .label4 {
+    grid-area: l4;
+  }
+
+  .input4 {
+    grid-area: i4;
+  }
+
+  .label5 {
+    grid-area: l5;
+  }
+
+  .input5 {
+    grid-area: i5;
+  }
+
+  .label6 {
+    grid-area: l6;
+  }
+
+  .textarea {
+    grid-area: ta;
+  }
+
+  .main-div {
+    display: flex;
+    flex-direction: row;
```

Comment:
This line is what's causing your Reservation page to be too wide. You are saying "put all the elements next to each other in a row" but you aren't controlling how wide these elements are, so they just take up as much space as they want - and that adds up to more than 100%.

Change to this `flex-direction: column` and you will already be much closer. Then you can tweak the position of the large leaf icon using some tricks like _negative_ margin-top to push it up a bit (not great practice but in this case it's just a little help and could be worth it)